src/library/objects/Quest.js

```diff
@@ -96,20 +96,28 @@ Quest Type:
 		}
 		return "";
 	};
-	
+
 	/* INCREMENT
 	Add one to tracking progress
 	@param {number} reqNum - index of counter type, mainly for Bw1. Default: 0
 	@param {number} amount - progress amount to be increased. Default: 1
 	@param isAdjustingCounter - if true, prevent recursively inc on specific quest itself
 	------------------------------------------*/
-	KC3Quest.prototype.increment = function(reqNum, amount, isAdjustingCounter){
+	KC3Quest.prototype.increment = function(reqNum=0, amount=1, isAdjustingCounter=false){
 		var self = this;
 		var isIncreased = false;
+		if (! ConfigManager.spCounterAdjust) {
+			if (this.tracking && this.isSelected()) {
+				if (this.tracking[reqNum][0] + amount <= this.tracking[reqNum][1]) {

```

Comment:
@sinsinpub I was angry because I am trying to be professional: you are letting a known buggy feature out and making **every user**, not just our devs and testers experimental animals.

I didn't really check how the impl works in detail, but all I know is it's been there for at least a year and no one has reported having a problem with it. so I just want the old trusty behavior back.